src/ResourceManager/IotCentral/Commands.IotCentral.Test/ScenarioTests/IotCentralTests.ps1

```diff
@@ -0,0 +1,125 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+
+#################################
+## IotCentral Cmdlets			   ##
+#################################
+
+$global:resourceType = "Microsoft.IoTCentral/IotApps"
+
+<#
+.SYNOPSIS
+Tests CRUD Lifecycle Management for IoT Central Apps.
+#>
+function Test-IotCentralAppLifecycleManagement{
+	# Setup
+	$rgname = Get-ResourceGroupName
+	$rname = Get-ResourceName
+	$subdomain = ($rname) + "subdomain"
+	$location = "westus" #Get-Location "Microsoft.IoTCentral" "IotApps"
```

Comment:
@cle5eland any reason that the commented-out code can't be used instead of the hard-coded `westus` value?

Reply:
No reason--fixed.